server-common/src/main/java/org/apache/gravitino/server/authentication/OAuth2TokenAuthenticator.java

```diff
@@ -126,6 +126,13 @@ public void initialize(Config config) throws RuntimeException {
     this.defaultSigningKey = decodeSignKey(Base64.getDecoder().decode(configuredSignKey), algType);
   }
 
+  @Override
+  public boolean supports(byte[] tokenData) {
```

Comment:
Could you have a better name?What does  this authenticator support?

Reply:
I think `supports` is enough based on the given context. 

like https://github.com/spring-projects/spring-security/blob/480c84a6e1e0f16aa5fbb0b1fc5373640d6c9a1a/core/src/main/java/org/springframework/security/authentication/AuthenticationProvider.java#L63

Reply:
SupportsXXXX will be better. Because we may add other SupportsXXX methods.

Reply:
ok, i will change it